### ISSUE TITLE
pip install cbadc does not install numpy.typing

### ISSUE THREAD

**Issue by hanspi42** [hammal/cbadc#24]
... and the Jupyter notebook `plot_c_digital_estimator.ipynb` fails with 

```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-2-3efef77e9336> in <module>
----> 1 from cbadc.utilities import compute_power_spectral_density
      2 import matplotlib.pyplot as plt
      3 from cbadc.utilities import read_byte_stream_from_file, \
      4     byte_stream_2_control_signal
      5 from cbadc.utilities import random_control_signal

~\Anaconda3\lib\site-packages\cbadc\__init__.py in <module>
      4 """
      5 from . import analog_signal
----> 6 from . import analog_system
      7 from . import digital_control
      8 from . import digital_estimator

~\Anaconda3\lib\site-packages\cbadc\analog_system.py in <module>
      8 """
      9 import numpy as np
---> 10 import numpy.typing as npt
     11 import scipy.signal
     12 from typing import Tuple, List, Union

ModuleNotFoundError: No module named 'numpy.typing'
```

**Comment by hammal** [comment]
Thanks a lot. Will get right on it!

**Comment by hammal** [comment]
Hopefully fixed the dependency issue, see #33.

**Comment by hammal** [comment]
Also changed the wording in the downsampling tutorial such that we now refer to it as

Prepending a Virtual Bandlimiting Filter
-------------------------------------------

... in front of the system. Note that this filter is a conceptual addition and not actually part of the physical analog system.

Regardless, this effectively suppresses aliasing since we now reconstruct a signal shaped by both the STF of the system in addition to a bandlimiting filter.
